Troubleshooting/README.md

```diff
@@ -2,5 +2,6 @@ Troubleshooting Guide
 ---------------------
 
 -  [gfid to path](./gfid-to-path.md)
+-  [heal-info-and-split-brain-resolution](./heal-info-and-split-brain-resolution.md)
```

Comment:
This link name can be anything you want, like may be `Heal info and split brain resolution`